source/_docs/guides/pingdom-uptime-check.md

```diff
@@ -0,0 +1,94 @@
+
+## What You’ll Need
+
+- At least a paid starter account on Pingdom. It's OK if you don't have one yet, this guide goes over the signup process.
+- A Drupal or Wordpress site on Pantheon to monitor.
+
+## Create your Pingdom account
+
+1.  On [Pingdom's homepage](https://www.pingdom.com), click **Sign Up** or if available, **Start your Free 14-Day Trial**.
+
+    ![Pingdom.com home page](/source/docs/assets/images/integrations/pingdom_hp.png)
+
+2.  Review and choose from available plans and select either monthly or discounted annual billing. For this guide, we selected the **Pingdom Started Account**. Plans can be upgraded at anytime.
+
+    ![Pingdom.com choose plan](/source/docs/assets/images/integrations/choose_plan.png)
+
+    Add your credit card information. If you chose the free trial, you have until the end of the trial time to try Pingdom before the card is billed. Once you've read and agreed to Pingdom's [services agreement](https://www.pingdom.com/legal/software-service-agreement), click on **START FREE TRIAL** to continue.
+
+    Now we will log in and set up monitoring. You can use the **SET UP YOUR MONITORING** button to go directly to the Pingdom dashboard.
+
+## Create a Basic Uptime Check
+
+1.  When you first load the Pingdom dashboard, it will try to guide you through adding your website, and creating checks. Because this guide explains how to manually create checks, we suggest you skip this.
+
+2.  The left navigation menu contains the different types of monitoring checks, reports, and a few other features. From the [main user dashboard](https://my.pingdom.com/dashboard) or from the left navigation's **Monitoring** > **Uptime** sub-menu, click on **Add uptime check**:
+
+    ![My.pingdom.com dashboard](/source/docs/assets/images/integrations/dashboard.png)
+
+3.  In the modal that opens, add the following information where indicated:
+
+    - **Name of Check:** An easily recognizable name for the check, e.g. “mysite.com Home Page.”
+
+    - **Check Interval:** How often Pingdom will check the site. While one minute is the minimum and acceptable to use, often a check every 5 minutes or more is adequate, and will allow for brief network throughput issues. Remember, these checks also will appear in web access logs, so too many checks may make it harder to debug other issues.
+
+    - **Check Type:** You can monitor several different things with Pingdom: email services, network components such as DNS or specific ports, or a website. Choose **Web**.
+
+    - **URL/IP:** On the **Required** tab, enter the URL for the website. If your site is using HTTPS, select that dropdown option. From the **Optional** tab, you can add user credentials and expected response text. This is very useful if you are using Varnish to cache a site; you can create a simple PHP script which queries the database and returns a specific value to determine if the site is functioning as expected.
```

Comment:
Perhaps something like:
> If you have a custom domain, we suggest using it here to ensure that all redirects are working. If you're only concerned with the uptime of the site/CMS itself, you can point to your pantheonsite.io domain.

Needs work, but if you wanna add something like that I'm all for it.